README.md

```diff
@@ -33,7 +33,7 @@ development.
 
 ### Setup
 1. Install [Go 1.13](https://golang.org/doc/install).
-2. `go get github.com/google/keytransparency/cmd/keytransparency-client `
+2. `GO111MODULE=on go get github.com/google/keytransparency/cmd/keytransparency-client`
```

Comment:
If the user has installed Go 1.13, as specified in step 1, then this shouldn't be necessary should it? Nonetheless, it won't hurt.